### PR TITLE
feat: TE7-T1022: Implement login screen improvement

### PR DESCRIPTION
**Pull request by jigar-s-simform**
Implement login screen UI improvement

### CODE REVIEW

File: app/components/custom-input/CustomInputStyles.ts [hunk 1]
```diff
@@ -20,7 +20,8 @@ const customInputStyles = StyleSheet.create({
   inputContainer: {
     display: 'flex',
     flexDirection: 'row',
-    justifyContent:'space-between',
+    justifyContent: 'space-between',
+    alignItems:'center',
```
Comment (amitkajwani-simformsolutions) [inline app/components/custom-input/CustomInputStyles.ts]: space after :

File: app/redux/auth/AuthSlice.ts [hunk 2]
```diff
@@ -51,12 +51,14 @@ const authSlice = createSlice({
...
           state.userDetails = action.payload;
+          state.isLoading = false
         },
       )
       .addCase(
         loginThunk.rejected,
         (state: Draft<InitialAuthStateType>, action: PayloadAction<unknown>) => {
-          state.error = action.payload as string; // typecasting to string because from rejectWithValue we are returning string
+          state.error = action.payload as string;
+          state.isLoading=false// typecasting to string because from rejectWithValue we are returning string
```
Comment (amitkajwani-simformsolutions) [inline app/redux/auth/AuthSlice.ts]: isLoading accepts boolean then why comment is added for type casting to string?

File: app/theme/Colors.ts [hunk 3]
```diff
@@ -11,5 +11,6 @@ export default {
   grey: '#767676',
   red: '#FF0000',
   blackWithOpacity: '#000000AA',
-  whiteWithOpacity: '#ffffffAA'
+  whiteWithOpacity: '#ffffffAA',
+  darkBlue:'#5193F2'
```
Comment (amitkajwani-simformsolutions) [inline app/theme/Colors.ts]: space after :

File: app/modules/login/LoginStyles.ts [hunk 4]
```diff
@@ -8,15 +8,15 @@ import {
 
 const styles = StyleSheet.create({
   mainContainer: {
-    backgroundColor: Colors.white,
-    paddingTop: verticalScale(50),
+    backgroundColor: Colors.darkBlue,
     flex: 1,
   },
   loginText: {
+    marginVertical:verticalScale(15),
```
Comment (amitkajwani-simformsolutions) [inline app/modules/login/LoginStyles.ts]: add space after :

**Review by amitkajwani-simformsolutions — CHANGES_REQUESTED** [review]

File: app/components/custom-input/CustomInputStyles.ts (hunk 1 above)
Comment (jigar-s-simform) [reply app/components/custom-input/CustomInputStyles.ts]: resolved

File: app/redux/auth/AuthSlice.ts (hunk 2 above)
Comment (jigar-s-simform) [reply app/redux/auth/AuthSlice.ts]: resolved

File: app/theme/Colors.ts (hunk 3 above)
Comment (jigar-s-simform) [reply app/theme/Colors.ts]: resolved

File: app/modules/login/LoginStyles.ts (hunk 4 above)
Comment (jigar-s-simform) [reply app/modules/login/LoginStyles.ts]: resolved

**Review by amitkajwani-simformsolutions — APPROVED** [review]

**Pull request merged**